oktapam/client/cloud_connection_test.go

```diff
@@ -0,0 +1,226 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/okta/terraform-provider-oktapam/oktapam/utils"
+)
+
+func TestValidateCloudConnectionName(t *testing.T) {
+	// testing invalid name regexp, underscore is not allowed
+	data1 := CloudConnection{
+		Name:     utils.AsStringPtr("test_test"),
+		Provider: utils.AsStringPtr("aws"),
+		CloudConnectionDetails: &CloudConnectionDetails{
+			AccountId:  utils.AsStringPtr("123456789012"),
+			ExternalId: utils.AsStringPtr("550e8400-e29b-41d4-a716-446655440000"),
+			RoleArn:    utils.AsStringPtr("arn:aws:iam::123456789012:role/role_name"),
+		},
+	}
+	result1 := isCloudConnectionDataValid(data1)
+	if result1 != true {
```

Comment:
you should be using: 

```
if !result {
...
}
```

we know `result` is a boolean and not nil - we don't have to account for JavaScript truthiness

there are other examples of this in this file - please change them a well

Reply:
Done.